python/pysmurf/client/tune/smurf_tune.py

```diff
@@ -379,8 +379,39 @@ def tune_band_serial(self, band, n_samples=2**19,
         self.log('Done with serial tuning')
 
         
+    def plot_tune_summary_multiprocess(self, band):
+        """
+        Opens a new thread to make tune_summary plots.  This 
+        uses the multiprocessing module.
+
+        Args:
+        -----
+        band (int) : The band to make the plots in.
+        """
+        # Get some useful values that are repeatedly plotted
+        ref_phase_delay = self.get_ref_phase_delay(band)
```

Comment:
@slacrherbst I can't seem to get these variables in my new thread, so I have to get them in my main thread. Other PVs I can request. Any idea why some you can, and others you cannot?